src/py/aspen/api_utils.py

```diff
@@ -0,0 +1,15 @@
+from sqlalchemy.orm import joinedload
+
+from aspen.database.models.usergroup import User
+
+
+def get_usergroup(session, user_id: str):
+    # since authentication is required to access view this will always return one
+    user = (
+        session.query(User)
+        .options(joinedload(User.group))
+        .filter(User.auth0_user_id == user_id)
+        .one()
+    )
+
+    return user
```

Comment:
You should have this return a semi-composed query, and where you're using this, you can grab the `one()` or join it to other things.

```suggestion
def get_usergroup_query(session, user_id: str):
    # since authentication is required to access view this will always return one
    return (
        session.query(User)
        .options(joinedload(User.group))
        .filter(User.auth0_user_id == user_id)
    )
```